src/tests/alttestproj/fileName.env

```diff
@@ -0,0 +1 @@
+FOO=bar
```

Comment:
Is this used anywhere?

Reply:
no, this was a whole copy of the testproj directory so I didn't pollute that env. I can remove it since it's not used in this case.